src/app.ts

```diff
@@ -21,7 +21,10 @@ import {TestRuns} from "./test-runs";
 // TestRuns.testPreProcessingChain();
 
 //Testing of WetManager
-TestRuns.testWetManager();
+//TestRuns.testWetManager();
```

Comment:
`TestRuns` is a mess right now. I've started using Mocca JS in my latest PR and will rewrite the `TestRuns` class into unit tests.